Resources/Prototypes/_Harmony/tags.yml

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Can add a comment that this is added to upstream `ClothingShoesGaloshes` entity for ducky galoshes implementation. Optional.